docs/modules/index.ts.md

```diff
@@ -35,6 +37,54 @@ export declare const getCodec: <A extends Sum.AnyMember>(cs: MemberCodecs<A>, na
 
 Added in v0.1.0
 
+## getCodecFromMappedNullaryTag
+
+Derive a codec for any given sum `A` in which all the constructors are
+nullary, decoding and encoding to/from the constructor tags via conversion
+functions. Consider instead `getCodecFromStringlyMappedNullaryTag` for
+stringly APIs.
+
+**Signature**
+
+```ts
+export declare const getCodecFromMappedNullaryTag: <A extends NullaryMember>() => <B>(
+  from: (x: unknown) => O.Option<Tag<A>>,
```

Comment:
Could `x` here have type `B` instead of `unknown`?

Reply:
Nope, that's where the codec's input (`I = unknown`) goes.